src/dataframerow/dataframerow.jl

```diff
@@ -212,6 +212,18 @@ Convert a [`DataFrameRow`](@ref) to a `NamedTuple`.
 """
 Base.copy(r::DataFrameRow) = NamedTuple{Tuple(keys(r))}(values(r))
 
+Base.NamedTuple(dfr::DataFrameRow) = copy(dfr)
```

Comment:
I am tempted to say that this should be considered the primary function,
and `copy` should call this.

While I find the _its a view, `copy` materializes a view_ not completely unreasonable (but maybe it shoukd be in the docstring),
this feels like the main operation.

Reply:
`copy` has a docstring. I will change the rest.

Reply:
Yeah, but copy's doc string should say why it is returning a named tuple.

Reply:
OK - can you please propose something that would be appropriate (I have learned that native speakers are usually better at writing doc stings 😄; thank you)